doc/colorscheme-primary.txt

```diff
@@ -1,54 +1,53 @@
-﻿*primary.txt*	Primary color scheme based on Google's colors
-                                                                       primary
+﻿*colorscheme-primary.txt*	Primary color scheme based on Google's colors
+                               *colorscheme-primary.txt* *colorscheme-primary*
```

Comment:
BTW, I can't for the life of me figure out why the `*colorscheme-primary.txt*` on the first line is ignored and this extra `*colorscheme-primary.txt*`. It also seems to not be showing up in `:help local-additions` for probably the same reason.